website/content/api-docs/system/inspect/router.mdx

```diff
@@ -0,0 +1,97 @@
+---
+layout: api
+page_title: /sys/internal/inspect/router - HTTP API
+description: >-
+  The '/sys/internal/inspect/router' endpoint focuses on viewing the contents of specific structures in the internal router subsystem.
+---
+
+# `/sys/raw`
+The `/sys/internal/inspect/router` is intended to inspect a specific internal subsytem for debugging purposes.
+It can be accessed with a root token or sudo priveleges.
+
+## Root
+
+This endpoint returns a list of router entries in the router's root tree.
+
+| Method | Path          |
+| :----- | :------------ |
+| `GET`  | `/sys/internal/inspect/router/root` |
+
+### Sample Request
```

Comment:
I think it would be nice to include sample responses for these commands. The sample responses are really helpful when I'm trying to understand what a command actually shows.

Reply:
Added this information in the current commit!

Reply:
Thanks!